puppet/modules/quickstack/manifests/load_balancer/neutron.pp

```diff
@@ -6,6 +6,7 @@
   $backend_server_addrs,
   $api_port         = '9696',
   $api_mode         = 'tcp',
+  $timeout         = '200s',
```

Comment:
I have no problem with allowing override of the timeout here (I have a wip patch for a BZ asking to configure this for another driver as well), but I would like to get a good default here.  I do not know if 200s is it, sounds a little large to me. Our load balancer default is currently 30s.  Any thoughts @rohara ?